components/formats-bsd/src/loci/formats/in/OMETiffReader.java

```diff
@@ -1056,7 +1063,14 @@ public MetadataStore getMetadataStoreForConversion() {
     int realSeries = getSeries();
     for (int i=0; i<getSeriesCount(); i++) {
       setSeries(i);
```

Comment:
These are technically "public", but are not part of `IFormatReader`--they are specific to `OmeTiffReader` and using them requires hairy casts to get access to them.  `getMetadataStoreForConversion` has zero public use inside bioformats (and omero) and `getMetadataStoreForDisplay` has a single user in `ImageInfo`.  The special mangling for display could be moved into `ImageInfo` or `MetadataTools`.

If any writer is dependent upon `BinData.BigEndian` in the absence of any BinData elements, I would suggest we should fix those writers.  The valid metadata should be usable in all cases, without the need for special-case mangling which makes it invalid.  The fact these methods exist and aren't in `IFormatWriter` is a strong indication that there's something not quite right and that this is a workaround.

If we do retain these methods, I'd suggest they could simply wrap `getMetadataStore` and return the valid store without mangling, and we can then move the logic elsewhere which mangles things for special purposes (or eliminate it entirely).